### PR TITLE
Mobile 2675

### PR DESCRIPTION
**Pull request by oristanovic**

### CODE REVIEW

File: ios/Classes/AirshipPlugin.m [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
+    [SwiftAirshipPlugin registerWithRegistrar:registrar];
+}
+
++ (void)load {
+    NSNotificationCenter *center = [NSNotificationCenter defaultCenter];
+       [center addObserverForName:UIApplicationDidFinishLaunchingNotification
+                                                         object:nil
+                                                          queue:nil usingBlock:^(NSNotification * _Nonnull note) {
+           
+           [SwiftAirshipPlugin takeOffWithLaunchOptions:note.userInfo];
```
Comment (rlepinski) [inline ios/Classes/AirshipPlugin.m:15]: Is this the right user info?

**Review by rlepinski — APPROVED** [review]
Could we bump versions of  the SDK to 16.1

File: ios/Classes/AirshipPlugin.m (hunk 1 above)
Comment (oristanovic) [reply ios/Classes/AirshipPlugin.m:15]: yes

**Comment by oristanovic** [comment]
> Could we bump versions of the SDK to 16.1

also bumped plugin version to 5.1

**Review by Ulrico972 — APPROVED** [review]

**Pull request merged**